tests/regression/roles/sonic_vlans/defaults/main.yml

```
@@ -38,54 +43,62 @@ tests:
       - vlan_id: 500
         description: "modified vlan500 description"
       - vlan_id: 501
+      - vlan_id: 503
+      # Odd case where delete sets things to default true
+        autostate: true
   # replace test cases started
   - name: test_case_04
     description: Replace VLANs
     state: replaced
     input:
-        - vlan_id: 500
-        - vlan_id: 502
-          description: "modified vlan502 description"
+      - vlan_id: 500
+        autostate: true
+      - vlan_id: 502
+        description: "modified vlan502 description"
```

It would be good to verify that a 'replace' play that modifies autostate from 'true' to 'false' works correctly.

```suggestion
      - vlan_id: 500
        autostate: true
      - vlan_id: 501
        autostate: false
      - vlan_id: 502
        description: "modified vlan502 description"
```

---

The requested case still isn't present. This could be accomplished by setting autostate to true for vlan 504 in test_case_02 and setting it to false in test_case_04.